beacon_node/src/cli.rs

```diff
@@ -155,6 +155,14 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .default_value("5052")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("http-allow-origin")
+                .long("http-allow-origin")
+                .value_name("ORIGIN")
+                .help("Set the value of the Access-Control-Allow-Origin response HTTP header")
```

Comment:
Do you think it might be helpful to add ` Put *, to allow any origin` to this help text? I'm not sure I'd know what to set it to.

Reply:
Added.  It won't do any harm.